packages/shared/src/utils/platform/platform.js

```diff
@@ -12,11 +19,26 @@ export const isMT5 = () =>
     /^\/mt5/.test(window.location.pathname) ||
     (/^\/(br_)/.test(window.location.pathname) && window.location.pathname.split('/')[2] === 'mt5');
 
+export const getPathname = () => {
```

Comment:
you can merge this function and getPlatformHeader  these two functions are almost the same

Reply:
I don't think this is a good idea because `getPlatformHeader` is used for `Platform Switcher` which returns the platform from `app_routing_history` whereas `getPathname` returns `Reports`/ `Cashier`, which is when I need to remove the notification.